config/robotframework_aio/release_items_RobotFramework_TestsuitesManagement.json

```diff
@@ -46,14 +46,12 @@
 
    "RELEASES"  : {
                    "0.9." : [
-                              "RobotFramework_TestsuitesManagement change 0.9 A",
-                              "RobotFramework_TestsuitesManagement change 0.9 B",
-                              "RobotFramework_TestsuitesManagement change 0.9 X"
+                              "RobotFramework_TestsuitesManagement change 0.9 New RobotFramework 6.1 adaptation; Maintenance of log output"
                             ],
                    "0.10." : [
-                               "RobotFramework_TestsuitesManagement change 0.10 A",
-                               "RobotFramework_TestsuitesManagement change 0.10 B",
-                               "RobotFramework_TestsuitesManagement change 0.10 X"
+                               "RobotFramework_TestsuitesManagement change 0.10 Improve dotdict feature",
```

Comment:
Hi Son,

please avoid this massive redundancy: "RobotFramework_TestsuitesManagement change 0.10".

The first version of this file was only a sample I used for testing. But under productive conditions there is no need to put the component name in every row.

Reply:
Hi Son,
important is the end-user perspective.
Means:
Incompatibities, new features or stabilization.

Please follow Holgers Mail: "Release Tool: lessions learned".

Please also update jsonprepro release info.

Thank you,
 Thomas